imjoy/plugin.py

```diff
@@ -203,7 +203,7 @@ def kill_plugin(engine, pid):
         logger.warning("Timeout, force quitting %s", pid)
         kill_plugin(engine, pid)
     finally:
-        return
+        return  # TODO: What is the idea behind the return inside finally?
```

Comment:
Do we need the `return` inside the `finally`?

Reply:
This was a quick way to avoid capture error. I think it would be nicer to capture the error and log it.

Reply:
Ok, I'll fix that in another PR. :+1: